docs/monitoring/monitoring-metricbeat.asciidoc

```diff
@@ -0,0 +1,127 @@
+[role="xpack"]
+[[monitoring-metricbeat]]
+=== Monitoring {kib} with {metricbeat}
+
+beta[] In 6.4 and later, you can use {metricbeat} to collect data about {kib} 
+and ship it to the monitoring cluster, rather than routing it through the 
+production cluster as described in <<monitoring-kibana>>. 
+
+image::monitoring/images/metricbeat.png[Example monitoring architecture]
+
+To learn about monitoring in general, see 
+{stack-ov}/xpack-monitoring.html[Monitoring the {stack}]. 
+
+. Enable the collection of monitoring data. Set 
+`xpack.monitoring.collection.enabled` to `true` on the production cluster. +
++ 
+--
+For example, you can use the following APIs to review and change this setting:
+
+[source,js]
+----------------------------------
+GET _cluster/settings
+
+PUT _cluster/settings
+{
+  "persistent": {
+    "xpack.monitoring.collection.enabled": true
+  }
+}
+----------------------------------
+
+For more information, see {ref}/monitoring-settings.html[Monitoring settings in {es}] 
+and {ref}/cluster-update-settings.html[Cluster update settings].
+--
+
+. Disable the default collection of {kib} monitoring metrics. +
++
+--
+Add the following setting in the {kib} configuration file (`kibana.yml`): 
+
+[source,yaml]
+----------------------------------
+xpack.monitoring.kibana.collection.enabled: false
+----------------------------------
+
+Leave the `xpack.monitoring.enabled` set to its default value (`true`). 
+
+For more information, see 
+<<monitoring-settings-kb,Monitoring settings in {kib}>>.
+--
+
+. {metricbeat-ref}/metricbeat-installation.html[Install {metricbeat}] on the 
+same server as {kib}.
+
+. Enable the {kib} module in {metricbeat}. +
++
+--
+For example, to enable the default configuration in the `modules.d` directory, 
+run the following command:
+
+["source","sh",subs="attributes,callouts"]
+----------------------------------------------------------------------
+metricbeat modules enable kibana
+----------------------------------------------------------------------
+
+Alternatively, you can enable module configs in the {metricbeat} configuration 
```

Comment:
I wouldn't mention enabling the configs in metricbeat.yml because we are trying to move users away from that (it's better on multiple levels if they use the configs defined in the modules.d directory.